idaes/core/property_meta.py

```diff
@@ -345,6 +345,7 @@ def _create_derived_units(self):
             raise PropertyPackageError(
                 "{} cannot determine derived units, as property package has "
                 "not defined a set of base units.".format(self.name))
+                # PYLINT-TODO: maybe str(self) or self.__class__.__name__?
```

Comment:
What is the specific pylint error/warning associated with this? I am guessing that either fix suggested here would be fine - this might be a case of me thinking that `self` was a Pyomo component (where `name` is defined) when it is in fact a metadata object.

Reply:
The error is the missing `name` attribute: `E1101: Instance of 'PropertyClassMetadata' has no 'name' member (no-member)`. If `self.name` not existing is actually expected, I'll go ahead and set it to `str(self)`.

Reply:
This is probably just me think that `name` was defined when it was not - `str(self)` should be fine.